src/client/components/reservations/AddReservation.js

```diff
@@ -0,0 +1,91 @@
+import React, { useContext } from "react";
+import { useEffect, useState } from "react";
```

Comment:
```suggestion
import React, { useEffect, useState, useContext } from "react";
```

You can add everything to the same import statement.

Reply:
🙈